INSTALL.md

```diff
@@ -9,23 +9,31 @@ To install subscription-manager, run
 sudo dnf install subscription-manager
 ```
 
+This is not necessary on RHEL installations as subscription-manager is pre-installed and registration is required to install any packages with DNF.
+
```

Comment:
While true, I am not totally sure this is adding any value. Instead, maybe we could just say that

```md
RHEL already comes with `subscription-manager` pre-installed.
```

if we want to mention RHEL.

> registration is required to install any packages with DNF.

suggestion:

````md
With `subscription-manager` present, you can get register the system with your Red Hat account by running

```bash
sudo subscription-manager register
```
````